src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+// Task.java
```

Comment:
Consider adding Javadoc comments so that it's easier for reviewers to look through.